logic/dataframes/edit_dataframe.py

```
@@ -0,0 +1,55 @@
+import polars as pl
+from typing import Literal, Optional, OrderedDict
```

Ruff (0.8.2)

`typing.OrderedDict` imported but unused

Remove unused import: `typing.OrderedDict`

(F401)